de.danielnoethen.butt.json

```diff
@@ -0,0 +1,116 @@
+{
+    "app-id": "de.danielnoethen.butt",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "24.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "butt",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-run/pipewire-0",
+        "--env=ALSA_CONFIG_PATH=",
+        "--device=dri",
+        "--filesystem=~/.buttrc",
+        "--filesystem=xdg-documents",
+        "--filesystem=xdg-download",
+        "--filesystem=xdg-music",
+        "--filesystem=xdg-videos",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.mpris.MediaPlayer2.*",
+        "--talk-name=org.gnome.Rhythmbox3"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/man",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "portaudio",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/PortAudio/portaudio/archive/refs/tags/v19.7.0.tar.gz",
+                    "sha256": "5af29ba58bbdbb7bbcefaaecc77ec8fc413f0db6f4c4e286c40c3e1b83174fa0"
+                }
+            ]
+        },
+        {
+            "name": "portmidi",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/PortMidi/portmidi/archive/refs/tags/v2.0.4.tar.gz",
+                    "sha256": "64893e823ae146cabd3ad7f9a9a9c5332746abe7847c557b99b2577afa8a607c"
+                }
+            ],
+            "cleanup": [
+                "/bin"
+            ]
+        },
+        {
+            "name": "fltk",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/fltk/fltk.git",
+                    "tag": "release-1.3.9",
+                    "commit": "a66273e72621887fcbd0ee5dca2dd133356d5d8a"
+                }
+            ]
+        },
+        {
+            "name": "butt",
+            "buildsystem": "simple",
+            "build-commands": [
+                "cd butt && ./configure --with-client --prefix=/app",
+                "cd butt && make",
+                "cd butt && make install",
```

Comment:
this should be `buildsystem` `autotools`

Reply:
Are you sure? there isnt a autogen in the source

Reply:
and?

Reply:
`Error: module butt: Can't find autogen, autogen.sh or bootstrap` 
Unless im missing something obvious?

Reply:
you can add `no-autogen: true`

Reply:
Oh i see, cool all updated